src/main/java/com/gluonhq/substrate/Constants.java

```diff
@@ -195,4 +195,8 @@
      */
     public static final String NATIVE_IMAGE_ARG_CLASSPATH = "-cp";
 
+    /**
+     * Package
+     */
+    public static final String DEFAULT_APP_VERSION = "1.0.0";
```

Comment:
The package constants are at ReleaseConfiguration by now. They should be all together at a single place. Move them here?